src/framework/preferences/js/Panels.js

```diff
@@ -150,8 +150,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         // in order to reset the dom binder when panels are in an iframe.
         // It can be be eliminated once we have the new renderer.
         var userJQuery = that.container.constructor;
-        var oldContainer = that.container;
-        that.container = userJQuery(that.container.selector, oldContainer[0].ownerDocument);
+        var context = that.container[0].ownerDocument;
+        var selector = that.container.selector;
+        that.container = userJQuery(selector, context);
```

Comment:
Yes, we file it after the fact. There needs to be a clear paper trail leading from this code that someone can follow in order to gain an understanding of it.